client/forest-manager/src/lib.rs

```diff
@@ -1,5 +1,11 @@
+mod error;
 pub mod in_memory;
 pub(crate) mod prove;
+pub mod rocksdb;
 pub mod traits;
-pub mod types;
 pub(crate) mod utils;
+
+#[cfg(test)]
+mod test_utils;
+
+const LOG_TARGET: &str = "forest_storage";
```

Comment:
Nit: just following the standard on the rest of the project:
```suggestion
const LOG_TARGET: &str = "forest-storage";
```